laterpay/application/Controller/Install.php

```diff
@@ -295,6 +295,39 @@ public function maybe_update_time_passes_table() {
             $wpdb->query( 'ALTER TABLE ' . $table . ' DROP title_color, DROP description_color, DROP background_color, DROP background_path;' );
         }
     }
+    
+    /**
+     * Changing options names for api urls
```

Comment:
Since API and URL are acronyms, they should be written in uppercase. Please add a "." to the end of the sentence.